sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/models/LeaseAction.java

```diff
@@ -4,24 +4,33 @@
 
 package com.azure.storage.file.datalake.models;
 
-import com.fasterxml.jackson.annotation.JsonCreator;
-import com.fasterxml.jackson.annotation.JsonValue;
-
-/** Defines values for LeaseAction. */
+/**
+ * Defines values for LeaseAction.
+ */
 public enum LeaseAction {
```

Comment:
Why is there a `LeaseAction` in `implementation.models` and another in `models`? Can one of them be deleted?